test-server/test-server.c

```diff
@@ -34,6 +34,7 @@
 #ifdef EXTERNAL_POLL
 #define poll WSAPoll
 #endif
+#include "../win32port/win32helpers/gettimeofday.h"
```

Comment:
No, because gettimeofday() is used in this module:

w:/libwebsockets/test-server/test-server.c: In function 'callback_http':
w:/libwebsockets/test-server/test-server.c:299:4: warning: implicit declaration of function 'gettimeofday' [-Wimplicit-function-declaration]
    gettimeofday(&tv, NULL);
    ^